test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -1755,18 +1767,29 @@ void ProtoConverter::fillFunctionCallInput(unsigned _numInParams)
 
 void ProtoConverter::saveFunctionCallOutput(vector<string> const& _varsVec)
 {
-	for (auto const& var: _varsVec)
+	constexpr auto numSlots = 10;
+	constexpr auto slotSize = 32;
+
+	for (const auto& var: _varsVec)
 	{
 		// Flip a dice to choose whether to save output values
 		// in storage or memory.
-		bool coinFlip = counter() % 2 == 0;
+		auto diceThrow = counter() % (m_evmVersion.supportsTransientStorage() ? 3 : 2);
```

Comment:
```suggestion
		unsigned diceThrow = counter() % (m_evmVersion.supportsTransientStorage() ? 3 : 2);
```

BTW, a three-sided dice? :)